Homework2

KristianK1 opened the pull request.

KristianK1 committed:
first homework

KristianK1 committed:
updates

KristianK1 committed:
started changing category label

KristianK1 committed:
drugi ispravak

KristianK1 committed:
one column for actor card

KristianK1 committed:
!stray comma

KristianK1 committed:
removed unused imports

KristianK1 committed:
favorites screen and movie details without scroll

KristianK1 committed:
favorites screen and movie details without scroll

KristianK1 committed:
movie detail scroll

KristianK1 committed:
movie detail scroll

KristianK1 committed:
started task3

KristianK1 committed:
started task3

KristianK1 committed:
task 3

KristianK1 committed:
task 3

KristianK1 committed:
task 4 but doesnt handle physical back button

KristianK1 committed:
task 4 but doesnt handle physical back button

KristianK1 committed:
newlines

KristianK1 committed:
minor fixes

KristianK1 committed:
Merge pull request #1 from KristianK1/homework1

first homework

KristianK1 committed:
Merge branch 'homework2' of https://github.com/KristianK1/android-vjestina-tmdb into homework2

pero18 commented:
This PR is incorrect since there are changes from first PR in it. Please fix this. If you will have troubles doing this you can contact someone on Teams to help you with it.

KristianK1 commented:
> This PR is incorrect since there are changes from first PR in it. Please fix this. If you will have troubles doing this you can contact someone on Teams to help you with it.

I think i fixed it (Homework2.2, in homework2.1 i squashed it)
